setup/android_build_env.sh

```diff
@@ -34,6 +34,22 @@ sudo apt install -y adb autoconf automake axel bc bison build-essential clang cm
     libncurses5-dev libsdl1.2-dev libssl-dev libtool libxml2 libxml2-utils lzma* lzop maven ncftp ncurses-dev \
     patch patchelf pkg-config pngcrush pngquant python python-all-dev re2c schedtool squashfs-tools subversion texinfo \
     unzip w3m xsltproc zip zlib1g-dev "${PACKAGES}"
+	
+# For all those distro hoppers, lets setup your git credentials
+GIT_USERNAME="$(git config --get user.name)"
+GIT_EMAIL="$(git config --get user.email)"
+if [[ -z ${GIT_USERNAME} ]]; then
+	echo "Github User name: "
+	read -r USERNAME
+	git config --global user.name "$USERNAME"
```

Comment:
`"${USERNAME}"` and maybe change it to be just `NAME`

Reply:
addressed